lib/matplotlib/figure.py

```diff
@@ -1142,66 +1168,114 @@ def add_axes(self, *args, **kwargs):
         a.stale_callback = _stale_figure_callback
         return a
 
+    @docstring.dedent_interpd
     def add_subplot(self, *args, **kwargs):
         """
-        Add a subplot.
+        Add a subplot to the figure.
 
         Call signatures::
 
            add_subplot(nrows, ncols, index, **kwargs)
            add_subplot(pos, **kwargs)
+           add_subplot(ax)
 
         Parameters
         ----------
         *args
             Either a 3-digit integer or three separate integers
             describing the position of the subplot. If the three
-            integers are R, C, and P in order, the subplot will take
-            the Pth position on a grid with R rows and C columns.
+            integers are *nrows*, *ncols*, and *index* in order, the
+            subplot will take the *index* position on a grid with *nrows*
+            rows and *ncols* columns. *index* starts at 1 in the upper left
+            corner and increases to the right.
+
+            A 3-digit integer, *pos*, can be used if all integers are less
+            than 10.
```

Comment:
*pos* is a three digit integer, where the first digit is the number of rows, the second the number of columns, and the third the index of the subplot.  i.e. ``fig.add_subplot(235)`` is the same as ``fig.add_subplot(2, 3, 5)``.  Note that all integers must be less than 10 for this form to work.